src_py/apiServer/networkComponents.py

```diff
@@ -0,0 +1,83 @@
+import json
+
+class NetworkComponents():
```

Comment:
What is the purpose of this networComponents.py? 
Some explanation about the class will be very useful.

Reply:
networkComponents is the same as the old networkMap - which you already reviewed.
Its purpose is to recieve the architecture file (which Erlang also uses) - and parse it for clients, sources, routers.......
It may be used in the future to create more elaborate topologies.